libs/ml/benchmark/ops/ops.cpp

```diff
@@ -2283,74 +2303,112 @@ void BM_MatrixMultiply_Forward(benchmark::State &state)
   inputs.emplace_back(std::make_shared<TensorType>(input_2));
   fetch::ml::ops::MatrixMultiply<fetch::math::Tensor<T>> matmul;
 
+  auto cost_and_shape      = matmul.ChargeForward({config.shape, config.shape});
```

Comment:
This should be
auto cost_and_shape      = matmul.ChargeForward({in1_shape, in2_shape});

Reply:
oops